packages/react/src/navigation/index.tsx

```diff
@@ -11,15 +11,31 @@ import {
   CdsNavigationItem as NavigationItem,
 } from '@cds/core/navigation';
 import '@cds/core/navigation/register';
-import { createComponent } from '../converter/react-wrapper';
+import { createComponent } from '@lit-labs/react';
+import * as React from 'react';
 
-export const CdsNavigation = createComponent('cds-navigation', Navigation, {
+export const CdsNavigation = createComponent(React, 'cds-navigation', Navigation, {
   onExpandedChange: 'expandedChange',
   onCdsMotionChange: 'cdsMotionChange',
 });
-export const CdsNavigationGroup = createComponent('cds-navigation-group', NavigationGroup, {
+export const CdsNavigationGroup = createComponent(React, 'cds-navigation-group', NavigationGroup, {
   onExpandedChange: 'expandedChange',
   onCdsMotionChange: 'cdsMotionChange',
 });
-export const CdsNavigationStart = createComponent('cds-navigation-start', NavigationStart);
-export const CdsNavigationItem = createComponent('cds-navigation-item', NavigationItem);
+export const CdsNavigationStart = createComponent(React, 'cds-navigation-start', NavigationStart);
+export const CdsNavigationItem = createComponent(React, 'cds-navigation-item', NavigationItem);
+
+/**
+ * Setting displayName on the prototype is a workaround for now.
+ * See: https://github.com/lit/lit/issues/2154
+ * Follow: https://github.com/lit/lit/pull/2155
+ */
+CdsNavigation.displayName = 'CdsNavigation';
+CdsNavigationGroup.displayName = 'CdsNavigationGroup';
+CdsNavigationStart.displayName = 'CdsNavigationStart';
+CdsNavigationItem.displayName = 'CdsNavigationItem';
+
+// Setting react version has to be done at component level since we now consume the @lit-labs/react package
+if ((window as any).CDS && !(window as any).CDS._react.version) {
```

Comment:
The `window` object in some cases will not exist. Do you think it is a good idea to assure the `window` object exist before executing next lines?

Reply:
The original code had it this way. I modified it from that to current form.